003_bandwidth_check/Makefile

```diff
@@ -1,9 +1,9 @@
 all: hw
 
 hw:
-	 nvcc -x cu -O3 -std=c++11 -arch=sm_86 -I/usr/local/cuda-12.3/bin/../include ./bandwidth_check.cu -o main -L/usr/local/cuda-12.3/bin/../lib64
+	 nvcc -x cu -O3 -std=c++11 -arch=sm_86 -I/usr/local/cuda-12.3/bin/../include ./bandwidth_check.cu -o bandwidth_check -L/usr/local/cuda-12.3/bin/../lib64
```

Comment:
Please change this back, I am using a convention to make every executable "main" so it is easy from a gitignore perspective.